std-bits/table/src/main/java/org/enso/table/data/column/operation/map/numeric/BigDecimalRoundOp.java

```diff
@@ -0,0 +1,54 @@
+package org.enso.table.data.column.operation.map.numeric;
+
+import java.math.BigDecimal;
+import org.enso.base.numeric.Decimal_Utils;
+import org.enso.table.data.column.builder.Builder;
+import org.enso.table.data.column.operation.map.MapOperationProblemAggregator;
+import org.enso.table.data.column.operation.map.TernaryMapOperation;
+import org.enso.table.data.column.storage.SpecializedStorage;
+import org.enso.table.data.column.storage.Storage;
+import org.enso.table.data.column.storage.type.BigDecimalType;
+import org.enso.table.error.UnexpectedTypeException;
+import org.graalvm.polyglot.Context;
+
+/** An operation rounding floating-point numbers. */
```

Comment:
This comment seems to be misaligned with the class.

Reply:
Done